dll/win32/devmgr/properties/advprop.cpp

```diff
@@ -1897,6 +1897,32 @@ AdvProcDetailsDlgProc(IN HWND hwndDlg,
     {
         switch (uMsg)
         {
+            case WM_CONTEXTMENU:
+            {
+                if ((HWND)wParam == GetDlgItem(hwndDlg, IDC_DETAILSPROPVALUE))
+                {
+                    WCHAR szColName[255];
+                    INT nSelectedItems = ListView_GetSelectedCount((HWND)wParam);
+                    HMENU hPopup = CreatePopupMenu();
+
+                    if (LoadString(hDllInstance, IDS_COPY, szColName, sizeof(szColName) / sizeof(szColName[0])))
+                    {
+                        AppendMenuW(hPopup, MF_STRING, IDS_MENU_COPY, szColName);
+
+                        if (nSelectedItems <= 0)
+                            EnableMenuItem(hPopup, IDS_MENU_COPY, MF_BYCOMMAND | MF_GRAYED);
+
+                        POINT pt;
+                        GetCursorPos(&pt);
+
+                        TrackPopupMenu(hPopup, TPM_LEFTALIGN, pt.x, pt.y, 0, hwndDlg, NULL);
+                        Ret = TRUE;
+                    }
+
+                    DestroyMenu(hPopup);
```

Comment:
You create the menu above, but then do the track-menu thing only if the LoadString call succeeded.
Wouldn't it be better to create/track/destroy it  instead _only_ if the LoadString call succeeded?
Alternatively, what would happen instead if you keep the code as it currently is, except that you call the TrackPopupMenu _outside_ of the `if (LoadString(...))` block (and just before calling DestroyMenu)?